### PR TITLE
Add an aria-label to the profile button to improve accessibility.

### PR DESCRIPTION
**Pull request by sabakhilji**
In order to improve accessibility add an aria-label to the profile button.

## Pull Request checklist

Please review the below PR requirements:

- Build was run locally and any changes were pushed
- Ensure that the title clearly describes the changes
- Issue to be referenced in the PR description rather than in the commits or PR title
- Clean commit history

### CODE REVIEW

**Comment by benalleng** [comment]
Thanks for this PR! it looks like you may have placed the aria-label in not quite the right spot.
It should be outside of the onClick bracket for both iterations
```js
<button
        onClick={() => setOpenSignInModal(true)}
        onClick={() => setOpenSignInModal(true)}
        aria-label="profile"
        className="text-grey-300 ml-4 h-10 w-10 cursor-pointer"
      >
```

File: components/Navbar/UserButton.tsx [hunk 1]
```diff
@@ -19,14 +19,14 @@ export default function UserButton() {
...
-        onClick={() => setOpenSignInModal(true)}
+        onClick={() => setOpenSignInModal(true) aria-label="profile"}
         className="text-grey-300 ml-4 h-10 w-10 cursor-pointer"
       >
         <Avatar avatar={user.avatar} size={30} />
       </button>
     ) : (
       <button
-        onClick={() => toggleModal(true)}
+        onClick={() => toggleModal(true) aria-label="profile"}
```
Comment (benalleng) [inline components/Navbar/UserButton.tsx:29]: same here the aria-label needs to be outside of the closing curly-bracket
`onClick={() => toggleModal(true)}
aria-label="profile"`

**Review by benalleng — CHANGES_REQUESTED** [review]
Thanks for this PR! Great job! I left a review of the small changes you will need to make to ensure the code will work

File: components/Navbar/UserButton.tsx (hunk 1 above)
Comment (sabakhilji) [reply components/Navbar/UserButton.tsx:29]: Hey, thanks @benalleng. Will change it as you mentioned :)

**Comment by ecurrencyhodler** [comment]
@sabakhilji I'm super excited to see your first pull request! Let's use a more descriptive title to inform other reviewers what your pull request is about. For example "Add an aria-label to the profile button to improve accessibility." Would you mind doing this with your 2nd PR on adjusting the modal background as well? https://github.com/saving-satoshi/saving-satoshi/pull/212

You can also leave a comment in the description of the PR with a link of the issue it is addressing. Github will automatically detect the url and tie the PR and the issue together.

And thanks ben for reviewing!

**Review by benalleng — APPROVED** [review]
tACK with Orca screen reader, Now recognizes account modal button as: "Profile, push-button"

**Comment by benalleng** [comment]
image
nice :+1:

**Pull request merged**